Recruit-Training/Basic-Recruit-Training/pointers/main.cpp

```diff
@@ -19,29 +20,35 @@ int main()
     int *p = &x;
 
     //TODO in the line below predict what what is going to be output
-    cout << "Predicted value of p: " /* << PLACE YOUR PREDICTION HERE*/ << endl;
+    cout << "Predicted value of p: 0xsomething" /* << PLACE YOUR PREDICTION HERE*/ << endl;
```

Comment:
You can remove the comment  `/* << PLACE YOUR PREDICTION HERE*/`. Do the same to all the other lines that this applies to.